database/migrations/2024_02_28_125358_create_stocks_table.php

```diff
@@ -15,6 +15,7 @@ public function up(): void
             $table->id();
 
             $table->integer('amount');
+            $table->integer('product_size_id');
             $table->integer('product_id');
```

Comment:
`product_size_id` en `product_id` zijn geen integers, maar FK, nu is de link niet goed gelegd in de database